modules/table/src/integrationTest/java/org/apache/ignite/internal/threading/ItKvRecordApiThreadingTest.java

```diff
@@ -207,6 +233,119 @@ private static RecordContext<Tuple> binaryRecordContext() {
         return new RecordContext<>(KEY_RECORD.toKeyTuple(), new Record(KEY, "one").toFullTuple(), new Record(KEY, "two").toFullTuple());
     }
 
+    @SuppressWarnings("rawtypes")
+    @CartesianTest
+    void commonViewFuturesCompleteInContinuationsPool(@Enum CommonViewAsyncOperation operation, @Enum ViewKind kind) {
+        CriteriaQuerySource tableView = kind.criteriaQuerySource();
+
+        @SuppressWarnings("unchecked") CompletableFuture<Thread> completerFuture = forcingSwitchFromUserThread(
```

Comment:
Same applies to this: if you replace some generics with a wildcard type, then no suppression is needed.